src/openassetio-core/src/managerApi/ManagerInterface.cpp

```diff
@@ -14,11 +16,29 @@ namespace managerApi {
 
 ManagerInterface::ManagerInterface() = default;
 
+#define UNIMPLEMENTED_ERROR(capability)                                                           \
+  fmt::format(                                                                                    \
+      "The '{}' method has not been implemented by the manager. Check manager capability for {} " \
+      "by calling `manager.hasCapability`.",                                                      \
+      __func__, ManagerInterface::kCapabilityNames[static_cast<int64_t>(capability)])
```

Comment:
[question] Don't wanna hold up the merge, but just curious why `int64_t` rather than `size_t` or `kCapabilityNames::size_type` or similar.  Semi-surprised there's no linter warning about signed conversions

Reply:
Hmm, there actually was a warning because I used `UL` initially, that being the direct type of the `std::array`, but the warning message told me to use a sized-type like this. I took the type directly from the message without thinking about it.

But yeah, thinking about it for a second, this is a bit of a silly cast, I'll change it.

Reply:
[64afc94](https://github.com/OpenAssetIO/OpenAssetIO/pull/1141/commits/64afc9424aa28a8d9d9d604ed1d2d02264687346)